Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+readTrusted 'Dockerfile_jenkins_ubuntu32'
+
+pipeline {
+  agent none
+
+  stages {
+    stage('64bit') {
+      agent {
+        docker {
+          image 'golang:1.9.2'
+        }
+      }
+
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxdb
+
+        cd /go/src/github.com/influxdata/influxdb
+        go get github.com/sparrc/gdm
+        gdm restore
+        """
+
+        sh """
+        cd /go/src/github.com/influxdata/influxdb
+        go test -parallel=1 ./...
+        """
+      }
+    }
+
+    stage('32bit') {
+      agent {
+        dockerfile {
+          filename 'Dockerfile_jenkins_ubuntu32'
+        }
+      }
+
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxdb
+
+        cd /go/src/github.com/influxdata/influxdb
+        go get github.com/sparrc/gdm
+        gdm restore
+        """
+
+        sh """
+        cd /go/src/github.com/influxdata/influxdb
+        go test -parallel=1 ./...
+        """
+      }
+    }
+
+    stage('tsi index') {
```

Comment:
We don't need a `tsi` specific runner. Where it makes sense to run tests on the `tsi` index they're already told to do so inside the test code.

Reply:
Same with this one. It's currently like that in the CircleCI tests. I can remove it though.

Reply:
Yep it can go now, I think.